flexget/plugins/output/download.py

```diff
@@ -155,9 +155,9 @@ def save_error_page(self, entry, task, page):
         received = os.path.join(task.manager.config_base, 'received', task.name)
         if not os.path.isdir(received):
             os.makedirs(received)
-        filename = os.path.join(received, '%s.error' % entry['title'].encode(sys.getfilesystemencoding(), 'replace'))
+        filename = os.path.join(received, '%s.error' % entry['title'].encode(sys.getfilesystemencoding(), 'replace').decode(sys.getfilesystemencoding()))
```

Comment:
The error came below. For this, I noticed that the filename shows up as:
path/to/b'entry title'.error
as opposed to (in Py2)
path/to/entry title.error

Encode then decode in this case would at least drop characters that are not valid in the file system encoding, which I'm guessing at some point fixed a bug. I do wonder though if maybe we shouldn't change the way the file name is generated. Perhaps using pathscrub?